README.md

```diff
@@ -210,6 +210,19 @@ A `socket` or `host` parameter can be specified to connect through a unix socket
 DATABASE_URL="postgres://username:password@/database_name?socket=/var/run/postgresql"
 ```
 
+For passwordless authentication like Postgres [peer auth-method](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html) the URL can be shortened:
+
+```sh
+DATABASE_URL="postgres://username/database_name?socket=/var/run/postgresql"
+```
+
+Additionaly, when the username and database name are identical the URL can be shortened further:
```

Comment:
Huh, this is also surprising to me. I guess it's postgres-specific behavior. It's not something dbmate does.

Reply:
Same, i tested it with Postgres 14.9.

Reply:
It actually has nothing to do with the username and database name being identical.  When relying on Postgres peer authentication method, specifying the username and hostname in the connection string are unnecessary:

Check this out:

```
postgres@e8f183dbe0dd:/tmp$ ./dbmate -u postgres://hostnameIsIgnoredWhenSocketIsUsed/dbmate_test?socket=/var/run/postgresql status
[ ] 20231115033218_test.sql

Applied: 0
Pending: 1

postgres@e8f183dbe0dd:/tmp$ ./dbmate -u postgres:///dbmate_test?socket=/var/run/postgresql status
[ ] 20231115033218_test.sql

Applied: 0
Pending: 1
```

I think adding these shorthand Postgres database URL examples to the documenation might be handy, but the current PR's explanation for why they work in the README are not accurate.